coreax/score_matching.py

```diff
@@ -143,7 +142,7 @@ def _tree_flatten(self):
 
         return children, aux_data
 
-    def __init__(
+    def __init__(  # noqa: PLR0913, PLR0917
```

Comment:
I thought we had updated the max arguments in the pylint config?

Reply:
We have, but I thought this specific case was on the side of unreasonable to be the limit for the whole project. I.E we have set a project wide value of 10, this case is 13/14.

Reply:
Oh I see what you mean. The values in the `.pylintrc` diverge from those I've set for ruff in `pyproject.toml`. I'll just set them to match and remove the noqa